doc/src/citing.md

```diff
@@ -30,11 +28,14 @@ A BibTeX entry for LaTeX users is
      url = {https://doi.org/10.7717/peerj-cs.103},
      doi = {10.7717/peerj-cs.103}
     }
+```
 
-SymPy is BSD licensed, so you are free to use it whatever you like, be it
+[SymPy is BSD licensed](../../LICENSE), so you are free to use it whatever you like, be it
```

Comment:
This doesn't work. You'll need to either link to the license file on GitHub, or use `.. literalinclude` like we did with the AUTHORS to copy it into the docs.

Reply:
Thanks, I thought that might be dicey. I'll link to the [LICENSE on GitHub](https://raw.githubusercontent.com/sympy/sympy/master/LICENSE).